systemtests/bin/upload_symbols.py

```diff
@@ -85,8 +83,8 @@ def upload_symbols(ctx, expect_code, auth_token, base_url, symbolsfile):
                 fg="yellow",
             )
         )
-        try:
-            with METRICS.timer("upload_time"):
+        with METRICS.timer("upload_time"):
+            try:
```

Comment:
I swapped these contexts so it prints an elapsed time even if it fails with an unhandled exception.